frontend/src/components/FAQ/Faqs.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react';
+import CollapsibleHeader from './CollapsibleHeader'
+
+type Props = {
+  readonly data: {
```

Comment:
Not sure why we need an extra layer of `data` here. It might be easier to just define this prop type as 
```
type Props = {
   readonly headerName: string;
   readonly faqs: {
      question: string;
      answer: string;
   }
}
```